src/main/java/ch/njol/skript/registrations/EventValues.java

```diff
@@ -125,98 +115,123 @@ public EventValueInfo(final Class<E> event, final Class<T> c, final Getter<T, E>
 	 * It is recommended to use {@link EventValues#getEventValueGetter(Class, Class, int)} or {@link EventValueExpression#EventValueExpression(Class)} instead of invoking this
 	 * method repeatedly.
 	 * 
-	 * @param e
-	 * @param c
-	 * @param time
+	 * @param e event
+	 * @param c type of getter
+	 * @param time the event-value's time

```

Comment:
Unfortunately, this argument is <code>int</code>. You'll need to provide some hints about acceptable values, otherwise Javadoc is not very useful.